project/cmake/modules/FindCrossGUID.cmake

```diff
@@ -18,7 +18,8 @@ if(ENABLE_INTERNAL_CROSSGUID)
                                     <SOURCE_DIR> &&
                                     ${CMAKE_COMMAND} -E copy
                                     ${CORE_SOURCE_DIR}/tools/depends/target/crossguid/FindCXX11.cmake
-                                    <SOURCE_DIR>)
+                                    <SOURCE_DIR>
+                      BUILD_BYPRODUCTS ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/lib/libcrossguid.a)
```

Comment:
maybe use a var for the lib here and in CROSSGUID_LIBRARIES below?
Same for dvd libs